packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -653,6 +657,13 @@ describe('ReactErrorBoundaries', () => {
       container,
     );
     expect(container.firstChild.textContent).toBe('Caught an error: Hello.');
+    expect(console.error).toHaveBeenCalledWith(
```

Comment:
Does our `toWarnDev()` matcher work for this case?

Reply:
I don't think so, see https://github.com/facebook/react/pull/15797#discussion_r289583960